data/json/items/melee.json

```diff
@@ -1834,6 +1834,22 @@
     },
     {
         "type":"GENERIC",
+        "id":"sharp_rock",
+        "symbol": "*",
+        "color": "light_gray",
+        "name":"sharp_rock",
```

Comment:
The "name" entry specifies the normally displayed name ingame, it doesn't really have to match the id of the item in question. You probably want sharp rock instead of sharp_rock